common/datavalues/src/arrays/string/mutable.rs

```diff
@@ -59,13 +59,20 @@ impl MutableArrayBuilder for MutableStringArrayBuilder {
     }
 }
 
-impl Default for MutableStringArrayBuilder {
+impl Default for MutableStringArrayBuilder<true> {
     fn default() -> Self {
         Self::new()
     }
 }
 
-impl MutableStringArrayBuilder {
+impl Default for MutableStringArrayBuilder<false> {
```

Comment:
Can we combine them into one?

Reply:
the compiler does not allow me to do so : (